src/vs/workbench/parts/terminal/electron-browser/terminalInstance.ts

```diff
@@ -569,6 +569,9 @@ export class TerminalInstance implements ITerminalInstance {
 			env['PTYCOLS'] = cols.toString();
 			env['PTYROWS'] = rows.toString();
 		}
+		if (!env['LANG']) {
```

Comment:
I think it's a good idea to have a fallback, but this would probably be better in `_getLangEnvVariable` instead, replacing the `n === 0` case :+1: